Ucommerce.Sitefinity.UI/packages.config

```diff
@@ -44,9 +44,9 @@
   <package id="Telerik.Sitefinity.Core" version="13.1.7400" targetFramework="net472" />
   <package id="Telerik.Sitefinity.Feather.Core" version="13.1.7400" targetFramework="net472" />
   <package id="Telerik.Sitefinity.Mvc" version="13.1.7400" targetFramework="net472" />
-  <package id="uCommerce" version="9.3.1.20275" targetFramework="net472" />
-  <package id="uCommerce.Core" version="9.3.1.20275" targetFramework="net472" />
-  <package id="Ucommerce.Sitefinity" version="9.3.1.20275" targetFramework="net472" />
+  <package id="uCommerce" version="9.4.1.21068" targetFramework="net472" />
+  <package id="uCommerce.Core" version="9.4.1.21068" targetFramework="net472" />
+  <package id="Ucommerce.Sitefinity" version="9.4.1.21068" targetFramework="net472" />

```

Comment:
This is a huge commit. And most files only changed because `Ucommerce.Sitefinity`, a dependency in packages.config, installed them. 

I'd love to not have all of those files and dirs inside the `Ucommerce.Sitefinity.UI/Ucommerce` dir under source control (twice). They're really part of the main Ucommerce repository. 

If we could remove the `Ucommerce.Sitefinity` package and keep the `Ucommerce` and `Ucommerce.Core` packages, it would make future maintenance so much easier.